sigs/cncf-sigs.md

```diff
@@ -21,6 +21,7 @@ Scale contributions by the CNCF technical and user community, while retaining in
 * Engage more communities and create an on-ramp to effective TOC contribution & recognition.
 * Reduce some project workload on TOC while retaining executive control & tonal integrity with this elected body.
 * Avoid creating a platform for politics between vendors.
+* Provide an on-ramp for community members to get involved with the technical oversight of CNCF projects. As part of this, SIGs are expected to actively nurture diverse participation.
```

Comment:
would "ladder" work better here instead of "on-ramp"? (we already use on-ramp in line 21 and this is more about showing progression - taking on more well defined responsibilities over time )

Reply:
thanks - changed to "ladder"